portfolio/src/main/webapp/script.js

```diff
@@ -135,4 +135,32 @@ function createMap() {
   homeMarker.addListener('click', () => {
     homeInfo.open(map, homeMarker);
   });
+}
+
+async function getLoginStatus() {
+
+  const response = await fetch('/login-status');
+  const status = await response.json();
+ 
+  // user is logged in, allow them to view comments
+  if(status === "true") {
+    document.getElementById('comment-container').style.visibility = 'visible';
```

Comment:
Sometimes the ternary operator (condition ? exprIfTrue : exprIfFalse) can replace if statement.

Reply:
will keep in mind for next version!